package.json

```diff
@@ -29,14 +29,15 @@
     "url": "https://github.com/ipfs/is-ipfs.git"
   },
   "scripts": {
+    "prepare": "aegir build --no-bundle",
     "test:node": "aegir test --target node",
     "test:browser": "aegir test --target browser",
     "test": "aegir test",
     "lint": "aegir lint && aegir lint-package-json",
     "release": "aegir release",
     "release-minor": "aegir release --type minor",
     "release-major": "aegir release --type major",
-    "build": "aegir build",
+    "prepublishOnly": "aegir build",
```

Comment:
why do we need this ?

Reply:
To publish a minified bundle?  Have removed it, shout if it's actually useful.

Reply:
Did I miss any policy change for our JS libs? Why would we NOT provide a prebuilt browser bundle?

It is documented in https://github.com/ipfs-shipyard/is-ipfs#in-the-browser-through-script-tag  so for now I've restored `prepublishOnly`  in 2a03f54